README.md

```diff
@@ -79,12 +79,33 @@ Example:
 cti validate
 ```
 
-##### cti pack
+### cti pack
 
-Packs the package into a bundle.
+Packs the package into a bundle. Valid package should be in current working directory (or directory specified by `--working-dir`).
 
 Example:
 
+
+```shell
+> cti pack --include-source --format zip --prefix output --output=sample-package.cti
+
+> ls output
+sample-package.cti
 ```
-cti pack
-```
+
+#### --include-source
+
+Includes the source files into the bundle. By default, the source files are not included.
+Hidden files (starting with a dot) are not included in the bundle.
+
+#### --format
+
+The format of the output bundle. Supported formats are `zip` and `tgz`. Default is `tgz`.
+
+#### --prefix
+
+The directory where the output bundle will be saved. Default is `.`.
```

Comment:
Why both `--prefix` and `--output`? Isn't just `--output` already enough?

Reply:
just a standard approach